controller/backing_image_data_source_controller.go

```diff
@@ -427,8 +424,6 @@ func (c *BackingImageDataSourceController) syncBackingImageDataSourcePod(bids *l
 		podNotReadyMessage = fmt.Sprintf("pod spec node ID %v doesn't match the desired node ID %v", pod.Spec.NodeName, bids.Spec.NodeID)
 	} else if pod.DeletionTimestamp != nil {
 		podNotReadyMessage = "the pod dedicated to prepare the first backing image file is being deleted"
-	} else if pod.Spec.Containers[0].Image != defaultImage {
-		podNotReadyMessage = "the pod image is not the default one"
 	} else {
```

Comment:
Should still keep the below code in `syncBackingImageDataSourcePod` to make sure the the existing BIDS pod handling correctly.

```go
	} else if pod.Spec.Containers[0].Image != c.bimImageName {
		podNotReadyMessage = "the pod image is not the default one"
	} 
```

Reply:
Kept the check statement. Thanks